aptos-move/framework/move-stdlib/src/lib.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright © Aptos Foundation
+
+// Copyright (c) The Diem Core Contributors
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use log::LevelFilter;
+use move_command_line_common::files::{extension_equals, find_filenames, MOVE_EXTENSION};
+use move_compiler::shared::NumericalAddress;
+use std::{collections::BTreeMap, path::PathBuf};
+
+#[cfg(test)]
+mod tests;
+pub mod utils;
+
+pub mod natives;
+
+const MODULES_DIR: &str = "sources";
+const NURSERY_DIR: &str = "nursery";
+const DOCS_DIR: &str = "docs";
+const NURSERY_DOCS_DIR: &str = "nursery/docs";
+const ERRMAP_FILE: &str = "error_description.errmap";
+
+const REFERENCES_TEMPLATE: &str = "doc_templates/references.md";
+const OVERVIEW_TEMPLATE: &str = "doc_templates/overview.md";
+
+pub fn unit_testing_files() -> Vec<String> {
+    vec![path_in_crate("sources/UnitTest.move")]
+        .into_iter()
+        .map(|p| p.into_os_string().into_string().unwrap())
+        .collect()
+}
+
+pub fn path_in_crate<S>(relative: S) -> PathBuf
+where
+    S: Into<String>,
+{
+    let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    path.push(relative.into());
+    path
+}
+
+pub fn move_stdlib_modules_full_path() -> String {
+    format!("{}/{}", env!("CARGO_MANIFEST_DIR"), MODULES_DIR)
+}
+
+pub fn move_stdlib_docs_full_path() -> String {
+    format!("{}/{}", env!("CARGO_MANIFEST_DIR"), DOCS_DIR)
+}
+
+pub fn move_nursery_docs_full_path() -> String {
+    format!("{}/{}", env!("CARGO_MANIFEST_DIR"), NURSERY_DOCS_DIR)
+}
+
+pub fn move_stdlib_errmap_full_path() -> String {
+    format!("{}/{}", env!("CARGO_MANIFEST_DIR"), ERRMAP_FILE)
+}
+
+pub fn move_stdlib_files() -> Vec<String> {
+    let path = path_in_crate(MODULES_DIR);
+    find_filenames(&[path], |p| extension_equals(p, MOVE_EXTENSION)).unwrap()
+}
+
+pub fn move_nursery_files() -> Vec<String> {
+    let path = path_in_crate(NURSERY_DIR);
+    find_filenames(&[path], |p| extension_equals(p, MOVE_EXTENSION)).unwrap()
+}
+
+pub fn move_stdlib_named_addresses() -> BTreeMap<String, NumericalAddress> {
+    let mapping = [("std", "0x1")];
+    mapping
+        .iter()
+        .map(|(name, addr)| (name.to_string(), NumericalAddress::parse_str(addr).unwrap()))
+        .collect()
+}
+
+pub fn build_doc(
```

Comment:
Is this moved or copied from somewhere else?

Reply:
Copied from the Move repo.

But we don't need this because we do it already as part of the build process (and somehow automatically for CI), as seen here: https://github.com/wrwg/aptos-core/blob/bf2fb554305a987108e38ca67d8c3a32a5e1e886/aptos-move/framework/src/built_package.rs#L189. So lets remove this.